lib/packwerk/generators/templates/packwerk

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# This file was auto-generated by Packwerk through `packwerk init`
+
+# Needs to be run in test environment in order to have test helper paths available in the autoload paths
+ENV["RAILS_ENV"] = "test"
+
+# Command line arguments needs to be duplicated because spring modifies it
+packwerk_argv = ARGV.dup
+
+begin
+  load(File.expand_path("spring", __dir__))
+rescue LoadError => e
+  raise unless e.message.include?("spring")
+end
+
+require File.expand_path("../config/environment", __dir__)
```

Comment:
One improvement that I'd like to keep from Rafael's PR is to start the app only when we need it.

`ApplicationLoadPaths` starts the app, so we don't need to start it here. I think you can just remove this line.